src/dojos/cd-dojo-details.vue

```diff
@@ -201,6 +201,15 @@
         }
         return facebookData;
       },
+      isCDFAdmin() {
+        return !!(this.user && this.user.roles && this.user.roles.indexOf('cdf-admin') > -1);
+      },
+      isDojoAdmin() {
+        return this.isCDFAdmin || UsersDojosUtil.hasPermission(this.usersDojos, 'dojo-admin');
```

Comment:
hasPerm could handle the isCDFAdmin ?

Reply:
`hasPerm` is looking at the UsersDojo, not the user itself where the cdf-admin role is defined.